howdoi/howdoi.py

```diff
@@ -66,22 +65,40 @@ def u(x):
     'bing': SCHEME + 'www.bing.com/search?q=site:{0}%20{1}',
     'google': SCHEME + 'www.google.com/search?q=site:{0}%20{1}'
 }
+
+BLOCK_INDICATORS = (
+        'form id="captcha-form"',
+        'This page appears when Google automatically detects requests coming from your computer network which appear to be in violation of the <a href="//www.google.com/policies/terms/">Terms of Service'
+        )
+
 STAR_HEADER = u('\u2605')
 ANSWER_HEADER = u('{2}  Answer from {0} {2}\n{1}')
 NO_ANSWER_MSG = '< no answer given >'
-XDG_CACHE_DIR = os.environ.get('XDG_CACHE_HOME',
-                               os.path.join(os.path.expanduser('~'), '.cache'))
-CACHE_DIR = os.path.join(XDG_CACHE_DIR, 'howdoi')
-CACHE_FILE = os.path.join(CACHE_DIR, 'cache{0}'.format(
-    sys.version_info[0] if sys.version_info[0] == 3 else ''))
+
+CACHE_DIR = appdirs.user_cache_dir('howdoi')
+CACHE_ENTRY_MAX = 128
```

Comment:
Should this be configurable through `os.environ.get`?

Reply:
Imo, no. I wouldn't make it any larger (the lib has some O(n) behaviour w disk access on prune), and I wouldn't make it much smaller or it won't be useful. Making a useful change requires understanding how the cache works. If I ever get around to forking+fixing/improving cachelib, we can expose it.